### PR TITLE
Juliaflag

### PR DESCRIPTION
**Pull request by Adrianzatreanu**

### CODE REVIEW

File: coalib/bears/requirements/JuliaRequirement.py [hunk 1]
```diff
@@ -7,25 +7,22 @@ class JuliaRequirement(PackageRequirement):
...
-        '"Pkg.add(\"Lint\")"'
+        'Lint'
         >>> pr.version
         '19.2'
-        >>> pr.flag
-        '-e'
 
         :param package: A string with the name of the package to be installed.
         :param version: A version string. Leave empty to specify latest version.
         :param flag:    A string that specifies any additional flags, that
```
Comment (hypothesist) [inline coalib/bears/requirements/JuliaRequirement.py]: Remove the doc

Comment (arafsheikh) [inline coalib/bears/requirements/JuliaRequirement.py]: Forgot to remove this?

Comment (Adrianzatreanu) [inline coalib/bears/requirements/JuliaRequirement.py]: Ah forgot, yes. Thanks!

**Comment by hypothesist** [comment]
ack https://github.com/coala-analyzer/coala/pull/2436/commits/3a6d00c5bb39bbb5bb5456ca159d5eb4f67fb675

**Comment by hypothesist** [comment]
@rultor merge

**Comment by Adrianzatreanu** [comment]
@rultor merge

**Pull request merged**